build/docker/intel-deviceplugin-operator.Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN cd cmd/operator; GO111MODULE=${GO111MODULE} go install; cd -
 RUN chmod a+x /go/bin/operator \
     && install -D /go/bin/operator /install_root/usr/local/bin/intel_deviceplugin_operator \
     && install -D ${DIR}/LICENSE /install_root/usr/local/share/package-licenses/intel-device-plugins-for-kubernetes/LICENSE \
-    && scripts/copy-modules-licenses.sh ./cmd/operator /install_root/usr/local/share/package-licenses/
+    && scripts/copy-modules-licenses.sh ./cmd/operator /install_root/usr/local/share/
 
 FROM scratch as final
 COPY --from=builder /install_root /
```

Comment:
Not changed in this PR, but let me ask anyway: what's the advantage of building a root filesystem (`/install_root`) and then publishing it in a custom layer?

The downsides are that it prevents reusing the same base layer among two different container images (assuming of course that both use an up-to-date Clear Linux as base - probably not relevant in practice) and strictly speaking means that you distribute Clear Linux (download from CDN, publish on Docker Hub). This affects your bill of materials.

Reply:
I've been trying to promote fully static builds based on scratch instead of this install_root thing. I understand the desire to do the license copying, but I would really leave it at that, if possible. We'd get smaller containers as well.

Reply:
> I've been trying to promote fully static builds based on scratch instead of this install_root thing.

So you don't want to install *any* OS files? That may not be sufficient even for a statically linked Go binary. Depending on what it does, it needs certain data files.

> I understand the desire to do the license copying, but I would really leave it at that, if possible.

I'm not sure whether this `COPY -from=builder /install_root /` is done because of license copying.

> We'd get smaller containers as well.

Comparing container sizes is tricky. You need to make assumptions about the other containers on a system to determine whether common layers are reused or specific to a single container.